### ISSUE TITLE
Odysee posts show up as videos

### ISSUE THREAD

**Issue by b-risk** [futo-org/grayjay-android#1248]
### What happened?

On the official 'Odysee' channel on Odysee, posts show up as regular videos making the contents of the posts not visible, only platform comments on the posts are visible.

### Grayjay Version

253

### What plugins are you seeing the problem on?

Odysee

### Plugin Version

38

### When do you experience the issue?

- [ ] While logged in
- [ ] While logged out
- [X] N/A

**Comment by Tartabarta** [comment]
Might be correlated to 1203?

**Comment by z2rec** [comment]
There is no support for Odysee (blog) posts yet and that is the reason why they are showing like that. I recommend closing the issue here and opening a new one [here](https://github.com/futo-org/grayjay-plugin-odysee/issues), asking for feature implementation.

**Comment by b-risk** [comment]
> There is no support for Odysee (blog) posts yet and that is the reason why they are showing like that. I recommend closing the issue here and opening a new one [here](https://github.com/futo-org/grayjay-plugin-odysee/issues), asking for feature implementation.

Yeah I think they will have to add support for it with their existing posts implementation like they have for Patreon, also about that repo they have actually made multiple of them and even accidentally leaked a couple unfinished plugins in them like for TikTok and NicoNico and they haven't documented them yet here so I don't think they are ready yet for people to post issues in them, they also still allow you to pick any plugin listed when you make an issue here so I think they want them all here for now until further notice.
